app/assets/javascripts/client/inventories/analysis_consensus_answer.html

```diff
@@ -0,0 +1,58 @@
+<div class="answers" ng-show="question.answersVisible">
+  <div class="supporting-material">
+    <div class="supporting-products">
+      <div class="supporting-list">
+        <div class="supporting-list-header">
+          <span>Supporting Products</span>
+          <product-entry-list-link product-entries="productEntries" resource="question.score.supporting_inventory_response"></product-entry-list-link>
+        </div>
+        <div class="supporting-list-contents">
+          <p ng-repeat="selectedProductEntry in analysisConsensusAnswer.selectedProductEntries()">{{selectedProductEntry.general_inventory_question.product_name}}</p>
+        </div>
+      </div>
+      <div class="notes">
+        <span class="notes-header">Notes</span>
+        <textarea
+          ng-model="question.score.supporting_inventory_response.product_entry_evidence"
+          placeholder="This is completed by a designated facilitator during one or more in-person meetings with the team that participated in the inventory.">
+        </textarea>
+      </div>
+    </div>
+    <div class="supporting-data">
+      <div class="supporting-list">
+        <div class="supporting-list-header">
+          <span>Supporting Data</span>
+          <inventory-data-entry-list-link inventory-data-entries="inventoryDataEntries" resource="question.score.supporting_inventory_response"></inventory-data-entry-list-link>
+        </div>
+        <div class="supporting-list-contents">
+          <p ng-repeat="selectedInventoryDataEntry in analysisConsensusAnswer.selectedInventoryDataEntries()">{{selectedInventoryDataEntry.name}}</p>
+        </div>
+      </div>
+      <div class="notes">
+        <span class="notes-header">Notes</span>
+        <textarea
+          ng-model="question.score.supporting_inventory_response.data_entry_evidence"
+          placeholder="This is completed by a designated facilitator during one or more in-person meetings with the team that participated in the inventory.">
+        </textarea>
+      </div>
+    </div>
+  </div>
+  <div class="score">
+    <div ng-repeat="answer in question.answers">
+      <span class="score-title">{{analysisConsensusAnswer.answerTitle(answer.value)}}</span>
+      <label class="score-checkbox scored-{{answer.value}}">
+        <fake-radio proxy-ng-model="question.score.value" proxy-ng-value="answer.value"></fake-radio>
+      </label>
+    </div>
+    <!-- Temporary hack for hiding "skip". -->
+    <div ng-if="0 == 1">
```

Comment:
Interesting question. In my opinion, which would be humble if not endorsed by Douglas Crockford, one should always use the strict equality operator in JavaScript. By default. Every time.

That said, I really don't think it's worth the change here, because:

0. Both objects under comparison are not variables, but integers. You can't redefine the integer `0` to be the string `derp`, no matter what you do in JavaScript. This means neither `0` or `1` above will ever undergo any typecasting, save for a hypothetical sun flare.
0. `==` defers to `===` if both comparables are of the same type. You'd think this proxying would have harrowing performance implications but, as it turns out, it doesn't. Look at the code below. It sure does tell us that `===` takes a mere 99% of what `==` does. However, this 1% amounts to only approximately 0.02 seconds, and that's with a billion comparisons.

```
> console.time("==");
> for (var i=0; i<1000000000; i++) { 0 == 1; }
> console.timeEnd("==");
> console.time("===");
> for (var i=0; i<1000000000; i++) { 0 === 1; }
> console.timeEnd("===");
< ==: 2070.465ms
< ===: 2051.804ms

> console.time("==");
> for (var i=0; i<1000000000; i++) { 0 == 1; }
> console.timeEnd("==");
> console.time("===");
> for (var i=0; i<1000000000; i++) { 0 === 1; }
> console.timeEnd("===");
< ==: 2042.784ms
< ===: 1998.043ms

> console.time("==");
> for (var i=0; i<1000000000; i++) { 0 == 1; }
> console.timeEnd("==");
> console.time("===");
> for (var i=0; i<1000000000; i++) { 0 === 1; }
> console.timeEnd("===");
< ==: 2013.570ms
< ===: 2003.153ms

> ((2051.804+1998.043+2003.153)/(2070.465+2042.784+2013.570))
< 0.9879514965269907
```